beacon-chain/core/electra/churn_test.go

```diff
@@ -20,7 +20,7 @@ func createValidatorsWithTotalActiveBalance(totalBal primitives.Gwei) []*eth.Val
 	vals := make([]*eth.Validator, num)
 	for i := range vals {
 		wd := make([]byte, 32)
-		wd[0] = params.BeaconConfig().ETH1AddressWithdrawalPrefixByte
```

Comment:
This seems unexpected. Why is it changed?

Reply:
Related to: https://github.com/ethereum/consensus-specs/pull/4020
In latest spec, consolidation is only allowed if validator has 0x02